LL/lagin.ttl

```diff
@@ -365,89 +366,37 @@
     a foaf:Person;
     irishRel:nomName "Baeth";
     rel:childOf <#Ailb-af8156b0>;
-    rdfs:comment "Find Fili mac Rosa cecinit".
+    rdfs:comment "Find Fili mac Rosa cecinit";
+    rel:Relationship <#Findubur>.
```

Comment:
I'll replace it with foaf:knows, as it really isn't clear what the relationship is meant to be in this instance.